Makefile

```diff
@@ -1345,6 +1347,18 @@ glance_kuttl: input openstack_crds deploy_cleanup mariadb mariadb_deploy keyston
 	make deploy_cleanup
 	make cleanup
 
+.PHONY: manila_kuttl_run
+manila_kuttl_run: ## runs kuttl tests for the manila operator,
+	kubectl-kuttl test --config ${MANILA_KUTTL_CONF} ${MANILA_KUTTL_DIR}
+
+.PHONY: manila_kuttl
+manila_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_deploy keystone keystone_deploy infra ceph manila manila_deploy ## runs kuttl tests for manila operator. Installs openstack crds and manila operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
```

Comment:
It would probably be a good idea to introduce the changes to run the tests in a namespace different than `openstack` here

Reply:
Thanks Joan, I am planning to make the changes once this PR merges.